### PR TITLE
changes to cloud functions

### PR DESCRIPTION
**Pull request by ux-AylonS**
Signed-off-by: AylonS <aylons@reblaze.com>

### CODE REVIEW

File: src/assets/DatasetsUtils.ts [hunk 1]
```diff
@@ -82,10 +82,8 @@ const titles: { [key: string]: string } = {
   'backends-singular': 'Backend Service',
   'report': 'Report',
   'ignore': 'Ignore',
-  'request0': 'Request Pre Reblaze',
-  'request1': 'Request Post Reblaze',
-  'response0': 'Response Pre Reblaze',
-  'response1': 'Response Post Reblaze',
+  'request1': 'Request Post',
```
Comment (Aviv-Galmidi) [inline src/assets/DatasetsUtils.ts:85]: just `Request`

File: src/assets/DatasetsUtils.ts [hunk 2]
```diff
@@ -82,10 +82,8 @@ const titles: { [key: string]: string } = {
...
+  'response0': 'Response Pre',
```
Comment (Aviv-Galmidi) [inline src/assets/DatasetsUtils.ts:86]: just `Response`

File: src/assets/RequestsUtils.ts [hunk 3]
```diff
@@ -5,9 +5,9 @@ import {HttpRequestMethods} from '@/types'
 const confAPIRoot = '/conf/api'
 const confAPIVersion = 'v3'
 const reblazeAPIRoot = '/reblaze/api'
-const reblazeAPIVersion = 'v1.0'
+const reblazeAPIVersion = 'v3'
 const dataLayerAPIRoot = '/datalayer/api'
-const dataLayerAPIVersion = 'v1.0'
+const dataLayerAPIVersion = 'v3'
```
Comment (Aviv-Galmidi) [inline src/assets/RequestsUtils.ts:10]: Check with @Freakachoo if this was changed in the API to v3 and if it needs to be changed

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/assets/DatasetsUtils.ts (hunk 1 above)
Comment (ux-AylonS) [reply src/assets/DatasetsUtils.ts:85]: not implemented in the backend yet

File: src/assets/RequestsUtils.ts (hunk 3 above)
Comment (ux-AylonS) [reply src/assets/RequestsUtils.ts:10]: @Freakachoo do we need to change the dataLayer version from v1.0 to v3 ???

**Review by Aviv-Galmidi — APPROVED** [review]

File: src/assets/RequestsUtils.ts (hunk 3 above)
Comment (ux-AylonS) [reply src/assets/RequestsUtils.ts]: confirmed v1.0 for now

**Pull request merged**